windows.js

```diff
@@ -129,7 +132,17 @@ async function installGCCTools(type, version) {
 // A subset of the MSYS2 base-devel group
 async function installMSYS2Tools() {
   const downloadPath = await common.measure(`Downloading msys2 build tools`, async () => {
-    let url = `${msys2GCCReleaseURI}/msys2-gcc-pkgs/msys2.7z`
+    let builderPlatform = null
+    if (os.arch() === 'x64') {
+      builderPlatform = 'windows-latest'
+    } else if (os.arch() === 'arm64') {
+      builderPlatform = 'windows-11-arm64'
```

Comment:
With enough time probably it will be another version as the min GH-hosted runner in windows-arm64 like `windows-12-arm64` or so.
So we should name that platform `windows-arm64`

Reply:
I was following the existing pattern of macos-latest vs macos-13-arm64, but I agree that since this is completely new, we can use a different pattern. Let me update this.

Reply:
Right, but in hindsight that was a mistake :sweat_smile: https://github.com/ruby/ruby-builder?tab=readme-ov-file#naming